_pages/user-experience/faq-content.md

```diff
@@ -15,9 +15,60 @@ sidenav:
+
+When you first integrate with Login.gov, we recommend adding frequently asked questions (FAQ) content to ensure that users are able to understand that your application uses Login.gov as a sign-in service.. The next few sections outline plain language for agencies to leverage when creating that content.
+
+### What is Login.gov?
+Login.gov is a sign in service that offers secure and private online access to government programs, such as federal benefits, services and applications. With a Login.gov account, you can sign in to multiple government websites with the same email address and password. Learn more about Login.gov.
+
+### Why is [YOUR APPLICATION] using Login.gov?
+[APPLICATION] chose Login.gov as their sign-in provider as a security measure. This service requires you to use an email address, password and two-factor authentication, sometimes referred to as multi-factor authentication, to sign-in. The use of two-factor authentication is an added layer of security to protect your private and sensitive information from hackers and password compromises.
+
+**[Identity Verification ONLY]** In addition to meeting the above requirements for multi-factor authentication, Login.gov asks users to upload a photograph of their state-issued ID and share their address, phone number and other personal information which is then verified against authoritative sources. Verifying your identity makes sure the right person has access to the right information. We are using Login.gov’s simple and secure process to keep your sensitive information safe.
+
+### How do I create a Login.gov account?
+Start by selecting sign-in on [APPLICATION home page or link]. You’ll automatically be taken to the Login.gov sign-in page. You can create an account or sign in if you already have a Login.gov account. Login.gov is only the sign-in service. Once you sign in, you’ll be directed to your [APPLICATION] account or profile.
+
+Login.gov requires you to use a multifactor authentication method. You can select to receive a code via SMS/Text or phone call or use an authentication application, government PIV/CAC card, face or touch unlock, security key or download backup codes. We encourage you to add more than one authentication method to your account to avoid getting locked out if you ever lose an authentication method. <a
+    class="usa-link usa-link--external"
+    rel="noreferrer"
+    target="_blank"
+    href="https://www.login.gov/help/get-started/authentication-options/"
+    >Learn more about the different authentication methods you can use</a>.
+
+### How do I sign in to [APPLICATION] after I create my Login.gov account?
+Every time you click on Sign In on the APPLICATION site, you’ll go to the Login.gov sign in page. Enter your Login.gov email address, password and use your two-factor authentication method. . Once you enter this information correctly, you’ll be signed in and automatically directed back to the APPLICATION site.
```

Comment:
Extra period after the second sentence.

Reply:
fixed in https://github.com/18F/identity-dev-docs/pull/416/commits/a54ff2f891831b7429c17240343d2d83cc41b378